### ISSUE TITLE
How can I run centerpoint demo?

### ISSUE THREAD

**Issue by RookieXwc** [open-mmlab/mmdetection3d#1449]
Thank you for your excellent work for mmdetection3d，but i have a problem:
Refer to https://github.com/open-mmlab/mmdetection3d/tree/master/configs/centerpoint，I run:
```
python demo/pcd_demo.py
demo/data/nuscenes/n008-2018-08-01-15-16-36-0400__LIDAR_TOP__1533151603547590.pcd.bin
configs/centerpoint/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus.py
checkpoints/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus_20201004_170716-a134a233.pth
```
output log is:
```
load checkpoint from local path: checkpoints/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus_20201004_170716-a134a233.pth
The model and loaded state dict do not match exactly

size mismatch for pts_voxel_encoder.pfn_layers.0.linear.weight: copying a param with shape torch.Size([64, 10]) from checkpoint, the shape in current model is torch.Size([64, 11]).
/home/xwc/pythonproject/mmdetection3d/mmdet3d/core/bbox/coders/centerpoint_bbox_coders.py:202: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.post_center_range, device=heat.device)
/home/xwc/pythonproject/mmdetection3d/mmdet3d/core/bbox/coders/centerpoint_bbox_coders.py:202: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.post_center_range, device=heat.device)
```
It seems that the [weights](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/centerpoint/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus_20201004_170716-a134a233.pth) don't match the [configuration file](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/centerpoint/centerpoint_02pillar_second_secfpn_circlenms_4x8_cyclic_20e_nus.py)?
and the vis result seems wrong:
![image](https://user-images.githubusercontent.com/44500103/165956280-804b76d6-53ac-4f39-acc6-95a0a1a3f09a.png)
Can you help me? thank you! @[VVsssssk](https://github.com/VVsssssk)

**Comment by Xiangxu-0103** [comment]
The model you used may be the version before refactoring. I think the newest model hasn't been updated yet.

**Comment by RookieXwc** [comment]
Thank you! Can you offer the newest model? weights in this page https://github.com/open-mmlab/mmdetection3d/tree/master/configs/centerpoint，time stamp is 2020

**Comment by Tai-Wang** [comment]
We will update the weights after #1450  is merged.

**Comment by Tai-Wang** [comment]
Fixed in #1450 .
